awswrangler/_distributed.py

```diff
@@ -111,7 +112,7 @@ def wrapper(*args: Any, **kw: Dict[str, Any]) -> Any:
     def register(cls, name: Optional[EngineLiteral] = None) -> None:
         """Register the distribution engine dispatch methods."""
         with cls._lock:
-            engine_name = cast(EngineLiteral, name or cls.get_installed().value)
+            engine_name = cast(EngineLiteral, name or WR_ENGINE or cls.get_installed().value)
```

Comment:
Perhaps it can be further simplified to:
```
engine_name = cast(EngineLiteral, name or cls.get().value)
```
as `cls.get` would return the os.env and then default to installed if None. Hope I am not missing an edge case though

Reply:
Good idea!